R/hydromad.R

```diff
@@ -134,20 +369,26 @@ logLik.hydromad <-
     val
   }
 
+
+#' @rdname hydromad.object
+#' @export
 deviance.hydromad <- stats:::deviance.lm
 
+
+#' @rdname hydromad.object
+#' @export
 print.hydromad <-
   function(x, digits = max(3, getOption("digits") - 3), ...) {
     cat("\n",
-      "Hydromad model with ", toString(deparse(x$sma)), " SMA",
-      " and ", toString(deparse(x$routing)), " routing:", "\n",
-      sep = ""
+        "Hydromad model with ", toString(deparse(x$sma)), " SMA",
+        " and ", toString(deparse(x$routing)), " routing:", "\n",
+        sep = ""
     )
     rx <- x$data
-    cat("Start = ", index2char(index(rx)[1], frequency(rx)),
-      ", End = ", index2char(index(rx)[NROW(rx)], frequency(rx)),
-      "\n",
-      sep = ""
+    cat("Start = ", zoo::index2char(index(rx)[1], frequency(rx)),
+        ", End = ", zoo::index2char(index(rx)[NROW(rx)], frequency(rx)),
```

Comment:
```suggestion
        ", End = ", index2char(index(rx)[NROW(rx)], frequency(rx)),
```
zoo:: shouldn't be needed?
Was this an accidental reversion?

Reply:
Actually, reverting this back throws an error when I run R CMD check locally, so I'd suggest keeping it here:

```
print.hydromad: no visible global function definition for ‘index2char’
Undefined global functions or variables:
index2char
```